js/src/components/contact-information/index.js

```diff
@@ -39,29 +33,16 @@ const description = (
 	</>
 );
 
-const mcTitle = __( 'Verify contact information', 'google-listings-and-ads' );
 const settingsTitle = __( 'Contact information', 'google-listings-and-ads' );
 
 /**
  * Renders a preview of contact information section,
- * or a <NoContactInformationCard> if contact informations are not saved yet.
+ * or a <NoContactInformationCard> if contact information are not saved yet.
```

Comment:
```suggestion
 * or a notice if contact information is outdated.
```

Note: `<NoContactInformationCard>` has already been removed by #1025.